akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -79,14 +79,14 @@ import akka.util.OptionVal
  */
 private[akka] object InboundEnvelope {
   def apply(
-    recipient:        InternalActorRef,
+    recipient:        OptionVal[InternalActorRef],
```

Comment:
very good change btw, this fits what I feel to be natural when having worked with compression and the dummy guy for a while. 👍